src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RADIUS_NUMBER = 10;
```

Comment:
The `MAX_RADIUS_NUMBER` constant name implies it's the maximum value for the radius, but in the `getDefaultFigure` method, it is being used as the bound for a random number generation, which is misleading. Consider renaming it to reflect its purpose more accurately or adjusting its usage.